src/Tizen.Multimedia.MediaPlayer/Player/AudioPitch.cs

```diff
@@ -0,0 +1,125 @@
+        /// <since_tizen> 6 </since_tizen>
+        public bool IsEnabled
+        {
+            get
+            {
+                Player.ValidateNotDisposed();
+                NativePlayer.IsAudioPitchEnabled(Player.Handle, out var value).
+                    ThrowIfFailed(Player, "Failed to get whether the audio pitch is enabled or not");
+                return value;
+            }
+
+            set
+            {
+                Player.ValidateNotDisposed();
+                Player.ValidatePlayerState(PlayerState.Idle);
+
+                NativePlayer.SetAudioPitchEnabled(Player.Handle, value).
+                    ThrowIfFailed(Player, "Failed to enable the audio pitch of the player");
+            }
+        }
+
+        /// <summary>
+        /// Gets the audio pitch value.
+        /// </summary>
+        /// <returns>The pitch value of an audio stream. The default value is 1.</returns>
+        /// <remarks>This function is used for audio content only.</remarks>
+        /// <exception cref="InvalidOperationException">The player is not in the valid state.</exception>
+        /// <exception cref="ObjectDisposedException">The player has already been disposed of.</exception>
+        /// <seealso cref="IsEnabled"/>
+        /// <seealso cref="SetValue(float)"/>
+        /// <since_tizen> 6 </since_tizen>
+        public float GetValue()
+        {
+            Player.ValidateNotDisposed();
+
+            if (Player.AudioPitch.IsEnabled == false)
+                throw new InvalidOperationException("A pitch is not enabled.");
+
+            NativePlayer.GetAudioPitch(Player.Handle, out var value).
+                ThrowIfFailed(Player, "Failed to get the audio pitch");
+
+            return value;
+        }
+
+        /// <summary>
+        /// Sets the audio pitch value.
+        /// </summary>
+        /// <param name="value">The pitch value of an audio stream. The default value is 1.</param>
+        /// <remarks>This function is used for audio content only.</remarks>
+        /// <exception cref="ObjectDisposedException">The player has already been disposed of.</exception>
+        /// <exception cref="ArgumentOutOfRangeException">
+        ///     <pramref name="value"/> is less than 0.5.
+        ///     -or-<br/>
+        ///     <paramref name="value"/> is greater than 2.0.<br/>
+        /// </exception>
+        /// <seealso cref="IsEnabled"/>
+        /// <seealso cref="GetValue()"/>
+        /// <since_tizen> 6 </since_tizen>
+        public void SetValue(float value)
+        {
+            Player.ValidateNotDisposed();
+
+            if (Player.AudioPitch.IsEnabled == false)
+                throw new InvalidOperationException("A audio pitch is not enabled.");
+
+            if (value < 0.5F || 2.0F < value)
+            {
+                throw new ArgumentOutOfRangeException(nameof(value), value, "Valid level is 1.0 to 10.0");
```

Comment:
We can just ignore out of range values or clamp to the range

Reply:
hmm.. i don't get it. do you mean getting and throwing error from native api?
(there is a mistake in log..)

Reply:
``` c#
   if (value < 0.5F)
       value = 0.5F;
   if (value > 2.0F)
       value = 2.0F;
```

Reply:
the error is returned in native API. and we guarantee the same operation with native one. so i think it would be better to return the same error.

Reply:
We do not want same behavior with native, because we want improve usability of C# API more than native API

and if we clamp the value into the range, native API dose not return error

Reply:
then.. we should have made all APIs like that when c# api released..
every APIs return Argument exception currently in player.
if i add like that, we would have two policies for a range without any obvious reasons.

Reply:
I think it is not a big issue, 
Usually only method throw Argument exception
we also guide a avoid a exception on property

So, app developer did't confused about it.

Reply:
I just wonder what is the better choice. 
User set the value as -1 and return without any exception. In this case, I'm worry about that user could be confused as it's working well. 
I agree with your opinion that user should check description before use method.
But AudioPitch allow just 0.5~2.0 value range, and I think out of range value is abnormal case and it's better to throw the exception for it.

It is method and it can throw any exception if needed.

Reply:
BTW, does the Tizen C# have a rule over MS's? 
Here's dotnet guideline. (https://docs.microsoft.com/en-us/dotnet/standard/design-guidelines/property)
  ✓ DO preserve the previous value if a property setter throws an exception.
  X AVOID throwing exceptions from property getters.

If the exception is permitted in case of setting the property, user also can notified what they are trying to set the invalid value.
FYI, setting volume level property of AudioManager does not clamp the value rather it throws error also.

Reply:
Ok, I understand that better throw exception in this case.
but the range was already checked by native layer and the range could be changed by platform.
so, I think that no need to check range in c# layer

Reply:
I think C# API can check the range to avoid useless P/Invoke call, if the range will not be changed any more.